### DISCUSSION TITLE
webgl for Android

### DISCUSSION THREAD

**Discussion by max54id** [community/community#8642] [Mobile]
My pc and ios web pages are perfect, but for Android gameplay! 
index.html If you use style="width:99vw; height: 98vh; margin: auto" ---Android don't work
If you use full screen or style="width:99vw; 70vh; margin:auto" Android work, but it is too small.
I need a big Windows for Android, and there is no full screen!
My resolution is 900px*1600px.
Help me, Plz.
My webgl: https://max54id.github.io/GWgame/
Picture 1=Android error description----Height: 98vh
Picture 2=My index.html
Picture 3 = ios system --- height: 98vh (I need this for Android)
Picture 4=Android system---Height: 70vh (working, but too small)

![1](https://user-images.githubusercontent.com/30610851/145528565-43cc695c-b022-440a-af5b-e5221456f50e.jpg)
![2](https://user-images.githubusercontent.com/30610851/145528579-472c147d-6520-458e-bbc1-3982f0f037e5.jpg)
![3](https://user-images.githubusercontent.com/30610851/145528588-b0503daa-8258-4e9a-b694-4b02d2520869.jpg)
![4](https://user-images.githubusercontent.com/30610851/145528597-c676d6c2-4c4d-4ce0-bfee-8c424372d857.jpg)

**Comment by jyrBum-gycpac-4gy** [comment]
ع
